gulpfile.js

```diff
@@ -119,6 +119,7 @@ var ugliyOptions = {
     }
 };
 
+// Apply lerna.json version to root package.json
```

Comment:
I think these comments are non-related to this PR.. see point 4.1 of the contribution guidelines.